examples/custom-template.rs

```diff
@@ -0,0 +1,26 @@
+use std::thread;
```

Comment:
Please use the new feature in an existing example, and please do it in a separate commit.

Reply:
Thanks, deleted the example for now. Will modify an existing example in another commit later.